packages/xo-web/src/common/xo/index.js

```diff
@@ -345,9 +345,8 @@ export const subscribeRoles = createSubscription(
 
 export const subscribeIpPools = createSubscription(() => _call('ipPool.getAll'))
 
-export const subscribeResourceCatalog = createSubscription(() =>
-  _call('cloud.getResourceCatalog')
-)
+export const subscribeResourceCatalog = ({ filters } = {}) =>
```

Comment:
This function was already used, are you sure you did not break its usage?

Reply:
`({ filters } = {})` this makes it always compatible

Reply:
Nope because the return value has changed.